packages/utils/src/Utils.ts

```diff
@@ -189,3 +189,19 @@ export async function runContractScript<T extends ContractFactory> (provider: Pr
   if (parsed == null) throw new Error('unable to parse script (error) response: ' + ret)
   return parsed.args
 }
+
+/**
+ * sum the given bignumberish items (numbers, hex, bignumbers)
+ */
+export function sum (...args: BigNumberish[]): BigNumber {
+  return args.reduce((acc: BigNumber, cur) => acc.add(cur), BigNumber.from(0))
+}
+
+/**
+ * calculate the maximum verification cost of a UserOperation.
+ * the cost is the sum of the verification gas limits, multiplied by the maxFeePerGas.
+ * @param userOp
+ */
+export function getUserOpMaxCost (userOp: UserOperation): number {
```

Comment:
This looks like Go code. Shouldn't we make `UserOperation` a class?

Reply:
yes, should pack all userop related utils (resemble UserOperationLib in solidity - but we don't have "using" in TS..)